code/src/java/pcgen/io/PCGVer2Creator.java

```diff
@@ -2662,14 +2618,14 @@ private static String createComment(String s)
 	 * @param target
 	 * @return temp bonus name
 	 **/
-	private String tempBonusName(final Object creator, Object target)
+	private static String tempBonusName(final Object creator, Object target)
 	{
 		final StringBuilder cb = new StringBuilder(100);
 
 		cb.append(IOConstants.TAG_TEMPBONUS).append(':');
 		if (creator instanceof CDOMObject)
 		{
-			final CDOMObject oCreator = (CDOMObject) creator;
+			final Identified oCreator = (CDOMObject) creator;
```

Comment:
Wait, you're converting from A stuff = (A) thing  to B stuff = (A) thing?  Why not B stuff = (B) thing?
(This is a really confusing signal to a reader)